packages/ember-runtime/lib/utils.js

```diff
@@ -89,7 +89,7 @@ export function isArray(obj) {
   Ember.typeOf(new Number(101));        // 'number'
   Ember.typeOf(true);                   // 'boolean'
   Ember.typeOf(new Boolean(true));      // 'boolean'
-  Ember.typeOf(Ember.makeArray);        // 'function'
+  Ember.typeOf(Ember.A);                // 'function'
```

Comment:
I thought this was showing the return type `'function'`. 

Do we need the examples:
 - `Ember.typeOf(Ember.A())   // 'array'`
 - `Ember.typeOf(Ember.get)   // 'function'`